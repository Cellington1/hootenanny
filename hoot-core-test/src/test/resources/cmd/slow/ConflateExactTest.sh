#!/bin/bash
set -e

mkdir -p test-output/cmd/ConlateExactTest
STATS_OUT=test-output/cmd/ConlateExactTest/boston-road-building-out

hoot conflate -D writer.include.debug=true test-files/BostonSubsetRoadBuilding_FromOsm.osm test-files/BostonSubsetRoadBuilding_FromOsm.osm $STATS_OUT.osm --stats > $STATS_OUT

hoot is-match $STATS_OUT.osm test-files/cmd/slow/ConflateExactTest/output.osm
