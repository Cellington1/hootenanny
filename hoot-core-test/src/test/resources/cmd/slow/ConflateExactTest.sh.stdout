16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 123) Tags do not match:
16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 136) < hoot:review:note = The feature pair met neither the threshold for a match, miss, or review (review by default).
16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 137) > hoot:review:note = BuildingMatch: Way:-718, Way:-2787 p: match: 0.55 miss: 0.45 review: 0 note: 
16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 123) Tags do not match:
16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 136) < hoot:review:note = The feature pair met neither the threshold for a match, miss, or review (review by default).
16:17:53.539 WARN  ...core/scoring/MapComparator.cpp( 137) > hoot:review:note = BuildingMatch: Way:-853, Way:-2521 p: match: 0.425 miss: 0.175 review: 0.4 note: 
